src/notifiers/telegram-notifier.js

```diff
@@ -0,0 +1,24 @@
+import {logger} from '../logger.js';
+import {TelegramBotApiClient} from '../telegram-bot-api-client.js';
+import {config} from "../config.js";
+
+const telegramBotClient = new TelegramBotApiClient(config.telegram.botURL, config.telegram.botToken);
+
+class TelegramNotifier {
+    constructor(config) {
+        this.config = config;
+        this.chatID = this.config.chatID;
+    }
+
+    notify(message) {
+        if (this.chatID) {
```

Comment:
I think we can do this check in the constructor, so if it's misconfigured we'll be able to know at startup time rather than finding it out later when sending a notification

Reply:
agree, the new approach is better. no need for this _if_ now.